src/main/java/org/gridsuite/study/server/networkmodificationtree/RootNodeInfoRepositoryProxy.java

```diff
@@ -126,11 +133,31 @@ public BuildStatus getBuildStatus(AbstractNode node) {
     }
 
     @Override
-    public void invalidateBuildStatus(AbstractNode node) {
+    public void invalidateBuildStatus(AbstractNode node, List<UUID> changedNodes) {
         RootNode rootNode = (RootNode) node;
         if (rootNode.getBuildStatus() == BuildStatus.BUILT) {
             rootNode.setBuildStatus(BuildStatus.BUILT_INVALID);
+            changedNodes.add(rootNode.getId());
             updateNode(rootNode);
         }
     }
+
+    @Override
+    public void handleExcludeModification(AbstractNode node, UUID modificationUuid, boolean active) {
```

Comment:
Is it necessary to do something in this method since no modifications are associated with this node ?

Reply:
PR has now been rebased on main branch, after having merged the refactoring PR https://dev.azure.com/gridsuite/MVP/_sprints/backlog/MVP%20Team/MVP/Sprint%2041?workitem=752